src/Tribe/Tickets.php

```diff
@@ -471,13 +471,35 @@ public function has_permission( $post, $data, $nonce_action ) {
 			}
 
 			if ( ! empty( $data['ticket_start_date'] ) ) {
-				$meridian           = ! empty( $data['ticket_start_meridian'] ) ? ' ' . $data['ticket_start_meridian'] : '';
-				$ticket->start_date = date( Tribe__Date_Utils::DBDATETIMEFORMAT, strtotime( $data['ticket_start_date'] . ' ' . $data['ticket_start_hour'] . ':' . $data['ticket_start_minute'] . ':00' . $meridian ) );
+				$start_hour     = ! empty( $data['ticket_start_hour'] ) ? $data['ticket_start_hour'] : '00';
```

Comment:
What dependency that it was complaining? I've never saw that one. Merge away I will check.